docs/docsite/rst/command_guide/cheatsheet.rst

```diff
@@ -35,20 +35,31 @@ See :ref:`ansible-playbook` for detailed documentation.
 ansible-galaxy
 ==============
 
-Installing a collection:
-
 .. code-block:: bash
 
-   ansible-galaxy collection install mynamespace.mycollection
+  ansible-galaxy role install <namespace.rolename>
+  ansible-galaxy collection install -r requirements.txt
```

Comment:
@BhattacharjeeSutapa `requirements.yml` would be more appropriate instead of `requirements.txt`.

Reply:
[L45](https://github.com/ansible/ansible-documentation/pull/1053/files#diff-e871a6e3f35ffb1fa39943a9dfc9d2b72649b2a0ef18055f833215f1ddf1923bR45) as welll.